index.html

```diff
@@ -44,5 +44,128 @@ <h2>I’am a Software Developer</h2>
         <a href="#"><img src="src/Social-Media.svg" alt="social-media" /> </a>
       </div>
     </section>
+
+    <main class="my-works">
```

Comment:
- [ ] The container of the works section is meant to be  wrapped with a `<section>` tag. I am suggesting that you replace the `main` tag with a `section` tag. This will improve your codes readability.

> Normally the `main` tag is used to envelop all elements in the html page between the `header` and `footer` sections.